spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -289,75 +314,102 @@ def call_with_params(params)
 
       end
     end
+  end
+
+  describe "update" do
+    let!(:update_user) { create(:user) }
```

Comment:
Just a nitpick, I don't think the `!` from these three `let`s are necessary, just relying on the late binding works fine.